packages/nc-gui-v2/composables/useProject.ts

```diff
@@ -76,11 +76,32 @@ export function useProject(projectId?: MaybeRef<string>) {
     await loadTables()
   }
 
+  async function updateProject(data: Partial<ProjectType>) {
+    if (unref(projectId)) {
+      _projectId = unref(projectId)!
+    } else if (projectType === 'base') {
+      return
+    } else {
+      _projectId = route.params.projectId as string
+    }
+
+    await $api.project.update(_projectId, data)
+  }
+
+  function getProjectMeta() {
+    try {
+      return typeof project.value.meta === 'string' ? JSON.parse(project.value.meta) : project.value.meta
+    } catch (e) {
+      return {}
+    }
+  }
```

Comment:
This could be changed to a computed property so the value gets cached for re-use
```js
const projectMeta = computed(() => {
 try {
      return typeof project.value.meta === 'string' ? JSON.parse(project.value.meta) : project.value.meta
    } catch (e) {
      return {}
    }
})
```